src/Kconfig

```diff
@@ -43,6 +43,13 @@ config GOLIOTH_FW_UPDATE
 
 if GOLIOTH_FW_UPDATE
 
+config GOLIOTH_FW_UPDATE_PACKAGE_NAME
+    string "Package name of the main firmware component"
+    default "main"
+    help
+      The Package Name used by the Golioth Cloud to indicate the binary to use for device firmware
+      update. This name must be no more than GOLIOTH_OTA_MAX_PACKAGE_NAME_LEN characters in length.
```

Comment:
Let's add a static assert to enforce this

Reply:
Great suggestion, thanks!

![image](https://github.com/user-attachments/assets/63cfb42b-5d03-4a9d-ba6d-fe1efc235ec8)